mac/Keyman4MacIM/Keyman4MacIM/KMDownloadKeyboard/KMDownloadKBWindowController.m

```diff
@@ -23,7 +23,11 @@ - (void)windowDidLoad {
     [super windowDidLoad];
     
     [self.webView setFrameLoadDelegate:(id<WebFrameLoadDelegate>)self];
-    [self.webView.mainFrame loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"https://secure.tavultesoft.com/prog/macosx/10/downloadkeyboards/"]]];
+    
+    NSString *version = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
+    NSString *url = [NSString stringWithFormat:@"https://keyman.com/go/macos/10.0/download-keyboards/?version=%@", version];
+    NSLog(@"KMDownloadKBWindowController opening url = %@, version = '%@'", url, version);
+    [self.webView.mainFrame loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:url]]];
 }
 
```

Comment:
I'm unable to get this branch to build locally. I see that it does build on TC, so apparently the problems I'm having are related to something I've messed up on my machine and not a problem in the codebase in the repo. Ideally I'd like to be able to build it locally so I can easily play around with the plist thing.

Reply:
Ugh. Happy to rubber 🦆 if that helps.

Reply:
So, it turns out it actually BUILDs locally. The real problem is in the deploy step, which of course never happens on TC. Tracking that down now...

Reply:
You know what? I experienced this as well, just forgot about it... I ended up running localdeploy.sh myself with the DerivedData/... path to deploy, which worked. So perhaps localdeploy.sh is being called with the wrong parameter?

Reply:
I finally fixed it - PR 592. Turns out it was caused by adding the IM unit test because that required a new parameter to xcodebuild that suppresses the copy of the app to the build folder where we're expecting it.